clcache.py

```diff
@@ -177,7 +180,7 @@ def removeObjects(self, stats, removedObjects):
                 rmtree(dirPath)
             stats.setCacheSize(currentSize)
 
-    def getManifestHash(self, compilerBinary, commandLine, sourceFile):
+    def getManifestHash(self, compilerBinary, commandLine, sourceFilePath):
```

Comment:
You're right, the renaming is not necessary at all. I just found it helping since we are dealing with a lot of "file" variables within this Python script. A file may refer to a [file handle](https://docs.python.org/2/library/stdtypes.html#bltin-file-objects) when reading, the file's content in binary format, the file's content as a unicode string or the path to a file. Pythons type system does not make it easy to differentiate those on the first glance.

But at the end of the day, it's a matter of taste. Just tell me which one you would like to have :)